### ISSUE TITLE
Error detection on address entry isn't working

### ISSUE THREAD

**Issue by kelseyah** [CallParty/CallParty#173]
<img width="247" alt="screen shot 2017-03-05 at 2 57 26 pm" src="https://cloud.githubusercontent.com/assets/3522339/23590768/1f03580a-01b4-11e7-90e0-32a60dfd3e07.png">

**Comment by kelseyah** [comment]
Thinks my Senator is Bill Nelson? lol 

<img width="247" alt="screen shot 2017-03-05 at 2 59 42 pm" src="https://cloud.githubusercontent.com/assets/3522339/23590788/670b0a58-01b4-11e7-8bec-6feabacea233.png">

**Comment by mhfowler** [comment]
my guess with this is that the geocoder does return something for 123 street. can look into this. but might be a v1.1 thing to change the address entry to like tell you what district was matched and tell you how to change it if its not right or something

**Comment by oshoham** [comment]
yeah, @mhfowler's guess is right, there is error detection for the address conversation, but the geocoding API that we use is pretty permissive about what it considers a valid address. +1 for the suggestion of a conversation flow that presents you with the district that was matched and allows you to correct it

**Comment by oshoham** [comment]
maybe we also could eventually support a "change my address" conversation that can be started whenever there's no active conversation with the bot?

**Comment by kelseyah** [comment]
yea the "change my address" is in the product doc but was punted for future... problem with presenting the district is that users have no idea if it's right. we can think about this for now and just hope people aren't putting in bad addresses

**Comment by kelseyah** [comment]
@oshoham is there an easy way for us to manually changes someone's address or district? basically if someone gets the wrong members and realizes it they'll reach out to let us know and I want to make sure the fix wouldn't be crazy tedious
